src/lifetimes/struct-lifetimes.md

```diff
@@ -2,7 +2,7 @@
 minutes: 5
 ---
 
-# Lifetimes in Data Structures
+# Struct Lifetimes
```

Comment:
Is this the right title? This also works on enums. Maybe something like "Lifetimes in User Defined Types"?

Reply:
Actually, during the course I thought that using a struct with a field named `document` (rather than a newtype) would make the explanations somewhat more natural. What do you think?

Reply:
> Is this the right title? This also works on enums. Maybe something like "Lifetimes in User Defined Types"?

I'll change it to "Lifetimes in Data Structures" which just barely fits in the sidebar.

> Actually, during the course I thought that using a struct with a field named `document` (rather than a newtype) would make the explanations somewhat more natural. What do you think?

That's fixed in https://github.com/google/comprehensive-rust/pull/2585.